vm/src/builtins/object.rs

```diff
@@ -35,7 +35,7 @@ impl PyValue for PyBaseObject {
 impl PyBaseObject {
     /// Create and return a new object.  See help(type) for accurate signature.
     #[pyslot]
-    fn tp_new(mut args: FuncArgs, vm: &VirtualMachine) -> PyResult {
+    fn tp_new(_cls: PyTypeRef, mut args: FuncArgs, vm: &VirtualMachine) -> PyResult {
```

Comment:
because `cls` is provided outside, args decomposition need to be removed.

Reply:
i removed line 40, and changed _cls to cls to use cls provided from outside.
now the panic with the length gone, but still panics. The Error message is
Error in sys.excepthook:
```
lost sys.stderr
RuntimeError: lost sys.stderr
Original exception was:
lost sys.stderr
Traceback (most recent call last):
  File "_frozen_importlib", line 1094, in __import__
  File "_frozen_importlib", line 1015, in _gcd_import
  File "_frozen_importlib", line 992, in _find_and_load
  File "_frozen_importlib", line 992, in _find_and_load
  File "_frozen_importlib", line 976, in _find_and_load_unlocked
  File "_frozen_importlib", line 687, in _load_unlocked
  File "_frozen_importlib", line 685, in _load_unlocked
  File "_frozen_importlib", line 678, in _load_unlocked
  File "_frozen_importlib", line 672, in _load_unlocked
  File "_frozen_importlib_external", line 779, in exec_module
  File "_frozen_importlib_external", line 907, in get_code
  File "_frozen_importlib_external", line 883, in get_code
  File "_frozen_importlib_external", line 513, in _classify_pyc
BaseException: bad magic number in 'encodings': b'6afe'

The above exception was the direct cause of the following exception:

RuntimeError: Could not import encodings. Is your RUSTPYTHONPATH set? If you don't have access to a consistent external environment (e.g. if you're embedding rustpython in another application), try enabling the freeze-stdlib feature
thread 'main' panicked at 'initialization failed; exception backtrace above', vm/src/vm.rs:808:13
stack backtrace:
   0: rust_begin_unwind
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:493:5
   1: std::panicking::begin_panic_fmt
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:435:5
   2: rustpython_vm::vm::VirtualMachine::_py_panic_failed
             at ./vm/src/vm.rs:808:13
   3: rustpython_vm::vm::VirtualMachine::expect_pyresult::{{closure}}
             at ./vm/src/vm.rs:830:37
   4: core::result::Result<T,E>::unwrap_or_else
             at /home/eldpswp99/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:821:23
   5: rustpython_vm::vm::VirtualMachine::expect_pyresult
             at ./vm/src/vm.rs:830:9
   6: rustpython_vm::vm::VirtualMachine::initialize
             at ./vm/src/vm.rs:380:9
   7: rustpython_vm::vm::Interpreter::new_with_init
             at ./vm/src/vm.rs:2122:9
   8: rustpython::run
             at ./src/lib.rs:95:18
   9: rustpython::main
             at ./src/main.rs:2:5
  10: core::ops::function::FnOnce::call_once
             at /home/eldpswp99/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5

```